### PR TITLE
Update ci versions

### PR DESCRIPTION
**Pull request by bidoubiwa**
- [ ] Changes in CI naming should be reflected in `settings/branches/main/`

### CODE REVIEW

**Comment by curquiza** [comment]
bors try

**Comment by curquiza** [comment]
Why is there also change about integration tests name in this PR? is this expected?

**Comment by bidoubiwa** [comment]
> Why is there also change about integration tests name in this PR? is this expected?
> You can put it in the same, but want to be sure this is expected.

Yes it is expected! Should have been in another Pr but I was a bit lazy hhehe

File: bors.toml [hunk 1]
```diff
@@ -1,3 +1,8 @@
-status = ['integration-tests', 'style-check', 'cypress-run']
+status = [
+  'style-check',
+  'integration-tests (Node.js 12)',
+  'integration-tests (Node.js 14)',
```
[the remark this answers is not in the document]
Comment (bidoubiwa) [reply bors.toml:4]: I dont confirm compatibility with node 16 in this PR!

**Review by curquiza — APPROVED** [review]
LGTM

Don't forget to change the tests required by GH in the settings ;)

**Comment by bidoubiwa** [comment]
bors merge

**Comment by curquiza** [comment]
> Don't forget to change the tests required by GH in the settings ;)

You should do it before merging your PR

**Comment by bidoubiwa** [comment]
bors try

**Comment by bidoubiwa** [comment]
bors merge

**Pull request merged**